apps/managedidentity/managedidentity_test.go

```diff
@@ -128,6 +128,74 @@ func Test_SystemAssigned_Returns_AcquireToken_Failure(t *testing.T) {
 	}
 }
 
+func Test_RetryPolicy_For_AcquireToken_Request(t *testing.T) {
+	t.Run("Testing retry policy with With One failure", func(t *testing.T) {
```

Comment:
Why run this as a subtest? Isn't it simply the main test? There are no other subtests